go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/oauth-apiserver
 
-go 1.13
```

Comment:
also in Dockerfile?

Reply:
yes, [the Dockerfile](https://github.com/openshift/oauth-apiserver/blob/master/images/Dockerfile.rhel7#L1) already points to `registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6`

Reply:
We really need something in build machinery to check these, also in .ci-operator.yaml soon.